python/lsst/sims/maf/plots/plotHandler.py

```diff
@@ -9,6 +9,29 @@
 
 __all__ = ['PlotHandler', 'BasePlotter']
 
+
+def none_min(x):
+    """
+    check for min in an array and ignore None values
+    """
+    newx = [val for val in x if val is not None]
+    if len(newx) > 0:
+        return min(newx)
+    else:
+        return 0.
+
+
+def none_max(x):
+    """
+    check for min in an array and ignore None values
+    """
+    newx = [val for val in x if val is not None]
+    if len(newx) > 0:
+        return max(newx)
+    else:
+        return 0.
```

Comment:
This is weird...

`np.min(1.0, None)`

works, but

`np.min([1.0, None])`

does not.

I guess you need to lose the `[ ]`